src/modules/70-pipeline/components/Volumes/Volumes.tsx

```diff
@@ -0,0 +1,372 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import React from 'react'
+import { FormInput, Text, Button, Container, MultiTypeInputType, Layout } from '@wings-software/uicore'
+import { get } from 'lodash-es'
+import { Color, FontVariation } from '@harness/design-system'
+import cx from 'classnames'
+import { FieldArray } from 'formik'
+import { FormMultiTypeCheckboxField } from '@common/components'
+import { useStrings, UseStringsReturn } from 'framework/strings'
+import css from './Volumes.module.scss'
+
+export interface AddConditionInterface {
+  key: string
+  operator: string
+  value: string
+}
+
+interface VolumesPropsInterface {
+  formik?: any
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232285)
```suggestion
  formik?: unknown
```